zio-http/src/main/scala/zhttp/service/server/ServerChannelInitializer.scala

```diff
@@ -1,19 +1,33 @@
 package zhttp.service.server
 
+import io.netty.handler.codec.http.HttpObjectDecoder.{
+  DEFAULT_MAX_CHUNK_SIZE,
+  DEFAULT_MAX_HEADER_SIZE,
+  DEFAULT_MAX_INITIAL_LINE_LENGTH,
+}
 import io.netty.handler.codec.http.HttpServerKeepAliveHandler
 import zhttp.core._
-import zhttp.service.{HTTP_KEEPALIVE_HANDLER, HTTP_REQUEST_HANDLER, OBJECT_AGGREGATOR, SERVER_CODEC_HANDLER}
+import zhttp.service._
 
 /**
  * Initializes the netty channel with default handlers
  */
 @JSharable
-final case class ServerChannelInitializer(httpH: JChannelHandler, maxSize: Int) extends JChannelInitializer[JChannel] {
+final case class ServerChannelInitializer(httpH: JChannelHandler, hhtpMfdH: JChannelHandler, maxSize: Int)
+    extends JChannelInitializer[JChannel] {
   override def initChannel(channel: JChannel): Unit = {
     channel
       .pipeline()
-      .addLast(SERVER_CODEC_HANDLER, new JHttpServerCodec)
+      // Without increasing the DEFAULT_MAX_CHUNK_SIZE uploads are rather slow. Probably because Netty is chunking up messages
+      // itself. With a file upload, that causes lots of overhead. A factor >8 (resulting in 65K chunks) is ignored so it seems.
+      .addLast(
+        SERVER_CODEC_HANDLER,
+        new JHttpServerCodec(DEFAULT_MAX_INITIAL_LINE_LENGTH, DEFAULT_MAX_HEADER_SIZE, DEFAULT_MAX_CHUNK_SIZE * 8),
+      )
```

Comment:
I think we should add this handler also conditionally similar to the `MULTIPART_FORMDATA_HANDLER`